templates/taxi/car_form.html

```diff
@@ -1,11 +1,10 @@
 {% extends "base.html" %}
-{% load crispy_forms_filters %}
 
 {% block content %}
   <h1>{{ object|yesno:"Update,Create" }} car</h1>
   <form action="" method="post" novalidate>
     {% csrf_token %}
-    {{ form|crispy }}
+    {{ form.as_p }}
 
     <input type="submit" value="Submit" class="btn btn-primary">
   </form>
```

Comment:
Consider adding a `Cancel` button here. This button should redirect the user to the previous page they were on. This provides a better user experience as it allows the user to easily navigate away from the form if they decide not to submit it.